Tests/BackendIntegrationTests/OfflineStoreKitIntegrationTests.swift

```diff
@@ -241,16 +243,23 @@ class OfflineStoreKit1IntegrationTests: BaseOfflineStoreKitIntegrationTests {
 
     @available(iOS 15.0, tvOS 15.0, watchOS 8.0, macOS 12.0, *)
     func testCallToGetCustomerInfoWithPendingTransactionsPostsReceiptOnlyOnce() async throws {
+        // forceRenewalOfSubscription doesn't work well, so we use this instead
+        if #available(iOS 16.4, *) {
```

Comment:
It looks like we have this code duplicated in several tests, could it be abstracted out to a common function in `BaseStoreKitIntegrationTests`?